great_expectations/agent/__init__.py

```diff
@@ -1,2 +1 @@
-from great_expectations.agent.agent import GXAgent
 from great_expectations.agent.run import run_agent
```

Comment:
Need to maintain `run_agent` so the script from `setup.py` works (i.e. `gx-agent` in CLI)

Reply:
Shouldn't need to do that.
We probably need to remove the `setup.py` `gx-agent` console command from this repo.